frontend/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.test.tsx

```diff
@@ -213,6 +214,59 @@ describe("StreamlitMarkdown", () => {
     )
     expect(wrapper3.props().isButton).toEqual(true)
   })
+
+  it("colours text properly", () => {
+    const colorMapping = new Map(
+      Object.entries({
+        red: themeColors.mdRed,
+        blue: themeColors.mdBlue,
+        cyan: themeColors.mdCyan,
+        teal: themeColors.mdTeal,
+        violet: themeColors.mdViolet,
+        orange: themeColors.mdOrange,
+      })
+    )
+    const backgroundMapping = new Map(
```

Comment:
Is it planned to support background coloring with different formats?

Reply:
@sf-gh-szerszen Let's remove all the stuff for background colors for now. We can tackle this in a separate follow-up project. The main blocker is that we'll first have to find new colors, as background colors should be lighter than foreground.